fastar/test_util.py

```diff
@@ -27,8 +27,11 @@ def check(thunk, atol=None, rtol=None):
   jtu.check_close(result, expected, atol, rtol)
 
 def random_box(rng, shape):
-  box_start = rng.randint(shape)
-  box_shape = rng.randint(1, np.subtract(shape, box_start - 1))
+  if np.any(np.less(shape, 0)):
+    raise ValueError
+  box_start = np.array([0 if s == 0 else rng.randint(s) for s in shape])
```

Comment:
You could use a more terse NumPy-ish expression here, like
```python
box_start = rng.randint(np.maximum(s, 1))
```
I don't have a strong opinion about this though.

Reply:
Ah true, will do.